bool PWM_on
uint8 DutyCycle
uint8 GPIO_PinNumber